generated/ruby/grpc-google-iam-v1/grpc-google-iam-v1.gemspec

```diff
@@ -0,0 +1,26 @@
+# -*- ruby -*-
+# encoding: utf-8
+
+Gem::Specification.new do |s|
+  s.name          = 'grpc-google-iam-v1'
+  s.version       = '0.6.8'
```

Comment:
Hm, I see, thanks. So `googleapis-common-protos` would be really exceptional it exceeds `1.3`.